test/testConstants/FinancialDetailsTestConstants.scala

```diff
@@ -938,15 +938,15 @@ object FinancialDetailsTestConstants {
       outstandingAmount = Some(BigDecimal(-500.00)),
       paymentLotItem = None,
       paymentLot = None,
-      originalAmount = Some(BigDecimal(800.00)),
+      originalAmount = Some(BigDecimal(-800.00)),
```

Comment:
changes in this file seems to be not relevant ( if you rollback them back test would not fail)